srcopsmetrics/entities/README.rst

```diff
@@ -0,0 +1,134 @@
+====================================
+Meta-information Indicators Entities
+====================================
+
+Entity Criteria
+===============
+
+See template.py for necessary functionality that has to be provided in each Entity implementation.
+
+
+Meta-information Indicators Metrics
+===================================
+For every repository we want to gather following metrics:
+
+
+Time to Close an Issue (TTCI)
+-----------------------------
+For every closed Issue in repository.
+
+
+Median Time to Close an Issue (MTTCI_TIME)
```

Comment:
why do you add TIME in the acronym? also below in other definitions?

Reply:
because it's median time to close issue in time (I think that the more accurate definition is the cumulative median), not a rolling window one.

should I delete the `_TIME` acronym @pacospace ?

Reply:
an acronym is made by the initial letter of the words, my comment is releated to the use of word acronym. If you add _TIME suffix is no more an acronym right?